packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -28,13 +29,19 @@ import {initPeerIdAndEnr} from "./initPeerIdAndEnr.js";
 
 const DEFAULT_RETENTION_SSZ_OBJECTS_HOURS = 15 * 24;
 const HOURS_TO_MS = 3600 * 1000;
+const EIGHT_GB = 8 * 1024 * 1024 * 1024;
 
 /**
  * Runs a beacon node.
  */
 export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void> {
   const {config, options, beaconPaths, network, version, commit, peerId, logger} = await beaconHandlerInit(args);
 
+  const heapSizeLimit = getHeapStatistics().heap_size_limit;
+  if (heapSizeLimit < EIGHT_GB) {
+    logger.warn("Node.js heap size limit is too low, consider increasing it with --max-old-space-size=8GB");
```

Comment:
```suggestion
    logger.warn("Node.js heap size limit is too low, consider increasing it with --max-old-space-size=8192");
```

Reply:
I wonder if there's a way to make this more intuitive. Just appending --max-old-space-size won't do anything.
Eg:
./lodestar beacon --network mainnet ....... --max-old-space-size=8192

Reply:
it will throw an error due to unknown arg, but we might wanna make it more obvious that it has to be set a node arg which a lot of the time is set via `NODE_OPTIONS`

Reply:
Right, `NODE_OPTIONS` could be mentioned here. Or a link to our docs (that would have to be written).

Reply:
This sounds like an ideal candidate to add to https://chainsafe.github.io/lodestar/faqs